app/javascript/packages/document-capture/context/service-provider.js

```diff
@@ -5,6 +5,7 @@ import { createContext } from 'react';
  *
  * @prop {string} name Service provider name.
  * @prop {string} failureToProofURL URL to redirect user on failure to proof.
+ * @prop {boolean} ial2Strict Whether liveness capture should be expected from the user.
```

Comment:
I like that we move this to the ServiceProvider, but FWIW I think calling it `isLivenessEnabled` was a more descriptive name, maybe we could could switch it to *required* to be more meaningful? ex `serviceProvider.isLivenessRequired`?

Reply:
> I like that we move this to the ServiceProvider, but FWIW I think calling it `isLivenessEnabled` was a more descriptive name

Yeah, changing it to `ial2Strict` was more an exercise in consistency, though I'd agree to your point that it's a bit more clear to name it to the effect of requiring liveness.

> maybe we could could switch it to _required_ to be more meaningful? ex `serviceProvider.isLivenessRequired`?

Not sure I totally follow what's being suggested here. The value is required, but notably we can have an absence of a service provider altogether (when going direct to `/verify`). This is made more unwieldy by the fact that liveness should be checked when a service provider is omitted, hence the final condition `serviceProvider?.ial2Strict !== false`.

I guess it could be made a little less cumbersome to invert this where the service provider context value is an opt-out. Then we could at least simplify slightly to `! serviceProvider?.isLivenessDisabled`

Reply:
"required" as in, the service provider is requiring a liveness check of the user (but required vs enabled is not the hill I die on, it was just an attempt to communicate the intent of the config a little better)

Re: inverting --- I think flipping the value to disabled would make that *harder* to track and understand across server and client? To handle of missing service providers, we use the Null Object Pattern in the ruby code, so theres "always" a service provider even if it's essentially a stub object. so maybe we could take a similar approach, which means we revert to property checks (ex `if serviceProvider.name` instead of our current `if serviceProvider`)?

Reply:
> "required" as in, the service provider is requiring a liveness check of the user (but required vs enabled is not the hill I die on, it was just an attempt to communicate the intent of the config a little better)

Oh! I missed that you were contrasting "enabled" and "required". Yes, I think that could help clarify. 

On the topic of stubbing a service provider, I think it ties back a bit to the discussion we had previous at https://github.com/18F/identity-idp/pull/4228#discussion_r492984176, though I think it's now a bit more compelling:

- Consistency with server-side
- Being able to set the default `isLivenessRequired: true` in the stubbed service provider

I'll see about implementing this.

Reply:
Renamed `ial2Strict` to `isLivenessRequired` in c303333.

Provided stubbed default service provider context in 04fdaf5.

Reply:
Big fan! Thanks!

Reply:
Oh, forgot one more, which was another nice side-effect of this change: General truthiness check on `isLivenessRequired` in 94dc020.